src/utils/logging.cc

```diff
@@ -23,7 +23,11 @@
 
 #include <stdlib.h>
 #include <sys/types.h>
+#ifdef _MSC_VER
+#include <io.h>
```

Comment:
Is <io.h> a standard header file in windows?
If not, which lib should be installed to get io.h?